File: client/iih_stub/iih.ci_stub/iih.ci.rcm/hospitalcontagiondto/d/HospitalContagionDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.rcm.hospitalcontagiondto.d
{
    /// <summary>
    /// HospitalContagionDTO 的摘要说明。
    /// </summary>
    public class HospitalContagionDTO : BaseDTO {

        public HospitalContagionDTO() {
 
        }

        /// <summary>
        /// 传染病报告卡主键
        /// </summary>
		public string Id_contagion {
            get { return getAttrVal<string>("Id_contagion",null); }
            set { setAttrVal<string>("Id_contagion", value); }
        }

        /// <summary>
        /// 患者信息
        /// </summary>
		public FArrayList Ls_patient {
            get { return getAttrVal<FArrayList>("Ls_patient",null); }
            set { setAttrVal<FArrayList>("Ls_patient", value); }
        }

        /// <summary>
        /// 传染病报告卡信息
        /// </summary>
		public FArrayList Ls_contagion {
            get { return getAttrVal<FArrayList>("Ls_contagion",null); }
            set { setAttrVal<FArrayList>("Ls_contagion", value); }
        }

        /// <summary>
        /// 性别附卡
        /// </summary>
		public FArrayList Ls_std {
            get { return getAttrVal<FArrayList>("Ls_std",null); }
            set { setAttrVal<FArrayList>("Ls_std", value); }
        }

        /// <summary>
        /// 梅毒附卡
        /// </summary>
		public FArrayList Ls_syphilis {
            get { return getAttrVal<FArrayList>("Ls_syphilis",null); }
            set { setAttrVal<FArrayList>("Ls_syphilis", value); }
        }

        /// <summary>
        /// 手足口附卡
        /// </summary>
		public FArrayList Ls_hfm {
            get { return getAttrVal<FArrayList>("Ls_hfm",null); }
            set { setAttrVal<FArrayList>("Ls_hfm", value); }
        }

        /// <summary>
        /// 乙肝附卡
        /// </summary>
		public FArrayList Ls_hepatitisb {
            get { return getAttrVal<FArrayList>("Ls_hepatitisb",null); }
            set { setAttrVal<FArrayList>("Ls_hepatitisb", value); }
        }

        /// <summary>
        /// 传染病分类
        /// </summary>
		public FArrayList Ls_crb {
            get { return getAttrVal<FArrayList>("Ls_crb",null); }
            set { setAttrVal<FArrayList>("Ls_crb", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_contagion", "Ls_patient", "Ls_contagion", "Ls_std", "Ls_syphilis", "Ls_hfm", "Ls_hepatitisb", "Ls_crb"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.rcm.hospitalcontagiondto.d.HospitalContagionDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/rcm/hospitalcontagiondto/d/HospitalContagionDTO.java
package iih.ci.rcm.hospitalcontagiondto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 传染病信息dto DTO数据 
 * 
 */
public class HospitalContagionDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 传染病报告卡主键
	 * @return String
	 */
	public String getId_contagion() {
		return ((String) getAttrVal("Id_contagion"));
	}	
	/**
	 * 传染病报告卡主键
	 * @param Id_contagion
	 */
	public void setId_contagion(String Id_contagion) {
		setAttrVal("Id_contagion", Id_contagion);
	}
	/**
	 * 患者信息
	 * @return String
	 */
	public FArrayList getLs_patient() {
		return ((FArrayList) getAttrVal("Ls_patient"));
	}	
	/**
	 * 患者信息
	 * @param Ls_patient
	 */
	public void setLs_patient(FArrayList Ls_patient) {
		setAttrVal("Ls_patient", Ls_patient);
	}
	/**
	 * 传染病报告卡信息
	 * @return String
	 */
	public FArrayList getLs_contagion() {
		return ((FArrayList) getAttrVal("Ls_contagion"));
	}	
	/**
	 * 传染病报告卡信息
	 * @param Ls_contagion
	 */
	public void setLs_contagion(FArrayList Ls_contagion) {
		setAttrVal("Ls_contagion", Ls_contagion);
	}
	/**
	 * 性别附卡
	 * @return String
	 */
	public FArrayList getLs_std() {
		return ((FArrayList) getAttrVal("Ls_std"));
	}	
	/**
	 * 性别附卡
	 * @param Ls_std
	 */
	public void setLs_std(FArrayList Ls_std) {
		setAttrVal("Ls_std", Ls_std);
	}
	/**
	 * 梅毒附卡
	 * @return String
	 */
	public FArrayList getLs_syphilis() {
		return ((FArrayList) getAttrVal("Ls_syphilis"));
	}	
	/**
	 * 梅毒附卡
	 * @param Ls_syphilis
	 */
	public void setLs_syphilis(FArrayList Ls_syphilis) {
		setAttrVal("Ls_syphilis", Ls_syphilis);
	}
	/**
	 * 手足口附卡
	 * @return String
	 */
	public FArrayList getLs_hfm() {
		return ((FArrayList) getAttrVal("Ls_hfm"));
	}	
	/**
	 * 手足口附卡
	 * @param Ls_hfm
	 */
	public void setLs_hfm(FArrayList Ls_hfm) {
		setAttrVal("Ls_hfm", Ls_hfm);
	}
	/**
	 * 乙肝附卡
	 * @return String
	 */
	public FArrayList getLs_hepatitisb() {
		return ((FArrayList) getAttrVal("Ls_hepatitisb"));
	}	
	/**
	 * 乙肝附卡
	 * @param Ls_hepatitisb
	 */
	public void setLs_hepatitisb(FArrayList Ls_hepatitisb) {
		setAttrVal("Ls_hepatitisb", Ls_hepatitisb);
	}
	/**
	 * 传染病分类
	 * @return String
	 */
	public FArrayList getLs_crb() {
		return ((FArrayList) getAttrVal("Ls_crb"));
	}	
	/**
	 * 传染病分类
	 * @param Ls_crb
	 */
	public void setLs_crb(FArrayList Ls_crb) {
		setAttrVal("Ls_crb", Ls_crb);
	}
}